src/abr/puffer_ttp.cc

```diff
@@ -54,6 +77,48 @@ void PufferTTP::normalize_in_place(size_t i, vector<double> & input)
   }
 }
 
+
+void PufferTTP::calculate_gaussian_values() {
+    double gaussian_coefficient = 1.0/(std_val_*sqrt(2.0*M_PI));
```

Comment:
- We use 2 spaces for indentation
- Add whitespaces around each operator (`/` and `*`)
```
double gaussian_coefficient = 1.0 / (std_val_ * sqrt(2.0 * M_PI));
```